app-admin/pprocm/pprocm-1.0-r3.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_PN="PProcM"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="ncurses-based program to monitor CPU, disk, network and memory usage"
+HOMEPAGE="http://www.fusedcreations.com/PProcM/"
+SRC_URI="http://www.fusedcreations.com/PProcM/${MY_P}.tar.gz"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="
```

Comment:
Does this actually have a configure script/build anything -> DEPEND too, or can we just make them RDEPEND?